src/components/DecisioningEngine/createDecisionHistory.js

```diff
@@ -10,15 +10,14 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-const QUALIFIED_EVENT_TYPE = "decisioning.propositionQualified";
+import { PropositionEventType } from "../Personalization/constants/propositionEventType";
```

Comment:
We don't allow importing between components in Alloy. If this needs to be shared between the two components, put it into `src/shared`.